lib/github-pages-health-check/cdns/fastly.rb

```diff
@@ -4,6 +4,13 @@ module HealthCheck
     # Instance of the Fastly CDN for checking IP ownership
     # Specifically not namespaced to avoid a breaking change
     class Fastly < CDN
+      # Fastly maps used by GitHub Pages.
+      HOSTNAMES = %w(
+        github.map.fastly.net
+        github.map.fastly.net.
+        sni.github.map.fastly.net
+        sni.github.map.fastly.net.
```

Comment:
Minor, but curious why you included the `.` in the array, and not as a `\.?` after the union?